### DISCUSSION TITLE
Code in my website

### DISCUSSION THREAD

**Discussion by Zanyprogressive** [community/community#77059] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I’m using Really Simple SSL on my website and it alerted me to 2 instances of JS code that use http to pull content from another domain and are affecting my ability to use https for everything. I know nothing about coding or JavaScript, but looking through the code gibberish, I found a URL that led me to a page here for a specific user. I don’t recognize the app it led me to by that person, but it appears to be a core JS library? I’m just trying to figure out where the code is coming on my site. This is the URL I found in the middle of all the code: https://github.com/zloirock/core-js. Can anyone help me to figure out what is using this code on my site? And/or if it’s really an issue like Really Simple SSL says it is? Thank you for your patience in dealing with a total idiot when it comes to all this!
This is what is showing on Really Simple SSL: Cannot be fixed automatically, as the mixed content is coming from an external domain. Contact the owner of the domain to update the CSS/JS file

**Comment by zoelabbb** [comment]
# What is Core-js ?

`core-js` is an open-source project that provides a polyfill implementation for ECMAScript features that may not be supported in all JavaScript environments. ECMAScript is the standard that serves as the foundation for the JavaScript programming language.

A polyfill is JavaScript code that provides an implementation of features or APIs that may not be available in all environments or web browsers. By using polyfills, developers can ensure that their JavaScript applications or scripts can run in various environments, even if those environments don't fully support all the latest features of ECMAScript.

`core-js` includes a variety of polyfills for ECMAScript features such as `Promise`, `Map`, `Set`, `Object.assign`, and many more. The use of `core-js` can assist developers in writing consistent JavaScript code that can run across different environments.

It's important to note that as ECMAScript versions evolve, some web browsers and JavaScript environments may already natively support new features without requiring polyfills. However, in certain cases, the use of polyfills is still necessary to support older environments or specific limitations.

[Core-js Repo](https://github.com/zloirock/core-js#usage)
